.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+on: [push, pull_request]
+name: Test
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        rvm:
+          - 2.6
+          - 2.5
+          - 2.4
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        env:
+          BUNDLE_GEMFILE: ${{ github.workspace }}/Gemfile
+        with:
```

Comment:
Does this automatically handle `gem update bundler` already? If so, feel free to submit. The rest LGTM.

Reply:
It does not update the `bundler` gem explicitly, but I've found that this wasn't needed — the `ruby/setup-ruby` task already installs the latest `bundler`. Using `bundle-cache: true` causes it to run `bundle install` automatically.

Reply:
Got it.